plutus-benchmark/nofib/test/formulaBudget.pir-readable.golden

```diff
@@ -39,7 +39,7 @@
     !equalsInteger : integer -> integer -> Bool
       = \(x : integer) (y : integer) ->
           ifThenElse {Bool} (equalsInteger x y) True False
-    ~`$fOrdInteger` : Ord integer
+    !`$fOrdInteger` : Ord integer
```

Comment:
Now we know that the dictionary constructor is pure, so it's safe to strictify this!